python/GafferImageTest/CatalogueTest.py

```diff
@@ -586,7 +586,7 @@ def testDeleteBeforeSaveCompletesWithScriptVariables( self ) :
 		# we didn't even try to do so we make it read only.
 		os.mkdir( baseDirectory )
 		os.chmod( baseDirectory, stat.S_IREAD )
-		directory = baseDirectory / "${script:name}" / "images"
+		directory = baseDirectory / s.context()["script:name"] / "images"
```

Comment:
I think this is breaking the intent of the test - the intent is to check that variable substitutions in `s["c"]["directory"]` are handled correctly.